code/modules/lighting/lighting_turf.dm

```diff
@@ -53,8 +53,7 @@
 	if (!lighting_object)
 		return FALSE
 
-	return !(luminosity || dynamic_lumcount)
-
+	return !(loc:area_has_base_lighting || luminosity || dynamic_lumcount)
```

Comment:
should just add a getter for a turf's effective lumin

Reply:
Eh, it's one place.